### PR TITLE
Producers crud

### PR DESCRIPTION
**Pull request by sauloperez**

### CODE REVIEW

File: engines/producers/app/controllers/producers/application_controller.rb [hunk 1]
```diff
@@ -1,4 +1,63 @@
 module Producers
-  class ApplicationController < ActionController::Base
```
Comment (sauloperez) [inline engines/producers/app/controllers/producers/application_controller.rb]: Fuuu

File: engines/producers/app/controllers/producers/api/v1/producers_controller.rb [hunk 2]
```diff
@@ -0,0 +1,63 @@
+module Producers
+  module Api
+    module V1
+      class ProducersController < ApplicationController
+        before_action :load_producer, only: [:show, :update, :destroy]
+
+        # GET /api/v1/producers
+        #
+        def index
+          # no op
```
Comment (enricostano) [inline engines/producers/app/controllers/producers/api/v1/producers_controller.rb]: Maybe we could respond with the providers that are suppliers in the groups of the user. Too much? :joy:

Comment (sauloperez) [inline engines/producers/app/controllers/producers/api/v1/producers_controller.rb]: http://stream1.gifsoup.com/view6/4207605/mind-blown-o.gif

Comment (sauloperez) [inline engines/producers/app/controllers/producers/api/v1/producers_controller.rb]: Yo, you want me dead??

Comment (enricostano) [inline engines/producers/app/controllers/producers/api/v1/producers_controller.rb]: LOOOL

**Pull request closed without merging**